koku/subs/subs_data_extractor.py

```diff
@@ -129,6 +135,9 @@ def get_resource_ids_for_usage_account(self, usage_account, year, month):
 
     def gather_and_upload_for_resource_batch(self, year, month, batch, base_filename):
         """Gather the data and upload it to S3 for a batch of resource ids"""
+        # TODO: Determine if the AWS & AZURE resource_id to usage acount
```

Comment:
Since we are iterating through the usage accounts above, I will likely add a filter to the subs_summary.sql just incase.